### DISCUSSION TITLE
Change very large model with fixed ar-scale placement in the world

### DISCUSSION THREAD

**Discussion by georgianamarcu** [google/model-viewer#3907] [Q&A] [answered]
Hi,

I have a house blender model with large dimensions, 16m high, 15m wide, and I wish to preserve the scale so that the user won't be able to scale it down. What i need is the user to be able to place it on an empty lot in the real world, but at a further away distance from them, because at the moment, the model, being so big, appears very close, so you'd have to walk back a few meters to see it entirely. Is there a way to place by default the model in AR like 10-15m further away from the user?

I have tried the camera-orbit and camera-target attributes, but I assume they do not affect the placement in AR, just the model-viewer staging. Maybe is there something I am missing, or is this something that do not pertains to model-viewer but rather to the Android/iOS viewers? 

Thank you ^_^

**Comment by elalish** [comment, marked as answer]
Yeah, first question is what device / AR mode are you using? You're correct that we only have control over WebXR mode, so for the others I'm afraid you're out of luck. For WebXR (set `ar-modes="webxr"` to be sure), would you mind showing a screen recording?

**Reply by georgianamarcu** [reply]
Thank you, unfortunately Scene Viewer is prioritized.
